code/src/components/Header.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+import moment from 'moment'
+import { ClearButton } from './ClearButton'
+import styled from 'styled-components'
+
+export const Header = () => {
+    const tasks = useSelector((state) => state.todos.items)
+
+    return (
+        <Wrapper>
+            <WrapperLeft>
+                <BoldText>Todo</BoldText>
+                <RegularText>{moment().format("MMM Do")}</RegularText>
+            </WrapperLeft>
+            <WrapperRight>
+                <RegularText>{tasks.length} Tasks</RegularText>
+                <ClearButton />
+            </WrapperRight>
+        </Wrapper>)
+}
+
+const Wrapper = styled.section`
```

Comment:
try to stick to having the styled components on top of the file like you did above